libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -401,7 +401,7 @@ protected virtual async Task<bool> ProcessEventAsync(SequenceContext sequenceCon
 
                         break;
 
-                    case AdaptiveEvents.EndOfActions:
```

Comment:
Why the change? And you added "endOfPlan" so "endOfActions" is still there. What's the difference

Reply:
There are two events:
1) The sequence of actions has ended in which case we want to surface ambiguity events.  This is EndOfPlan and primariliy internal.
2) There are no more ambiguity events and we want to surface EndOfActions to possibly do prompts.

---
In reply to: [363905063](https://github.com/microsoft/botbuilder-dotnet/pull/3226#discussion_r363905063) [](ancestors = 363905063)